### DISCUSSION TITLE
[Question] Use segmented control as a native ios group

### DISCUSSION THREAD

**Discussion by sem4phor** [tamagui/tamagui#1673] [General]
Hello, I just learned about the segmented control component on ios. There is a react native package for that [here](https://github.com/react-native-segmented-control/segmented-control). Would it make sense to use it for horizontal Groups on ios when passing the native prop like in other tamagui components?

**Comment by natew** [comment]
Looks like a decent fit, would be a nice contribution. Moving to discussion but lmk if you'd like to try it.
